templates/github/quarkus-backend/template.yaml

```
@@ -58,7 +58,8 @@ spec:
       required:
         - groupId
         - artifactId
-        - javaPackageName
+        - version
+        - java_package_name
```

```suggestion
        - javaPackageName
```

Please use camelcase.